lambda_code/tests/test_mesh.py

```diff
@@ -76,21 +83,69 @@ def setUp(self):
     @mock_s3
     def test_write_to_s3(self):
```

Comment:
What's the purpose of this test? If we want to make sure that the content is written back, then we need to mock the `MeshOutputHandler` and assert if the write report has been called.

Reply:
I think this one is yours